tox.ini

```diff
@@ -514,7 +514,7 @@ basepython=python
 # DEV: We use `conftest.py` as a local pytest plugin to configure hooks for collection
 [pytest]
 # --cov-report is intentionally empty else pytest-cov will default to generating a report
-addopts = --cov=ddtrace/ --cov=tests/ --cov-append --cov-report= --durations=10
+addopts = --cov=ddtrace/ --cov=tests/ --cov-append --cov-report= --durations=10 --junitxml=test-results/junit.xml
```

Comment:
Wasn't 100% sure about adding here, but we need for tox and riot, this is the best/most common place for that.

Reply:
makes sense here imo